semantic-search/pom.xml

```diff
@@ -54,6 +49,17 @@
             <artifactId>json</artifactId>
             <version>20210307</version>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.9.1</version>
+            <scope>test</scope>
```

Comment:
Thanks again for teaching me we can just this one dependency rather than importing all the jupiter stuff individually!

Reply:
<3